.github/ISSUE_TEMPLATE/issue-content-develop.yml

```diff
@@ -0,0 +1,38 @@
+name: "📝 Developer Documentation"
```

Comment:
And I think we need to consider the ambiguity of this title, as many of our users will think this applies to them.

What about:

```suggestion
name: "📝 Documentation on develop.sentry.dev"
```